tests/framework/extensions/clusters/aks/importing/import_aks_cluster.go

```diff
@@ -0,0 +1,100 @@
+package importing
+
+import (
+	"context"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/cloudcredentials"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters/aks"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters/aks/resources"
+	"github.com/sirupsen/logrus"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+)
+
+// ImportAKSCluster is a helper function that imports an AKS cluster into Rancher. It will first create a resoure group
+// and then create a cluster in that resource group. It will then import the cluster into Rancher.
+func ImportAKSCluster(client *rancher.Client, cloudCredential *cloudcredentials.CloudCredential, clusterName string) (*management.Cluster, error) {
+	logrus.Infof("======================================")
+	logrus.Infof("TEST CASE: Importing AKS Cluster")
```

Comment:
Test cases should never be in extensions. Extensions to facilitate resource set up.